datafusion/functions-aggregate/Cargo.toml

```diff
@@ -39,6 +39,7 @@ path = "src/lib.rs"
 
 [dependencies]
 arrow = { workspace = true }
+arrow-schema = { workspace = true }
```

Comment:
for `downcast_integer`

Reply:
is the issue that not re-exported in arrow?

Reply:
yes, some macro has inevitable `arrow-schema` dependency https://github.com/apache/arrow-rs/issues/5676.